test/e2e/autoscale_test.go

```diff
@@ -39,6 +40,7 @@ import (
 
 const (
 	autoscaleExpectedOutput = "399989"
+	containerConcurrency = 1
```

Comment:
Why reduce this?

Reply:
To make the test lighter.  Is a higher value necessary?